docs/Bed_Mesh.md

```diff
@@ -372,14 +372,15 @@ are identified in green.
 
 ### Adaptive Meshes
 
-Adaptive bed meshing is a way to speed up the bed mesh generation by only probing
-the area of the bed used by the objects being printed. When used, the method will
-automatically adjust the mesh parameters based on the area occupied by the defined
-print objects.
+Adaptive bed meshing is a way to speed up the bed meshing process as well as waste less mesh information by only probing
+the area of the bed that will be used for printing. When used, this method will
```

Comment:
> When used, this method will [...]

What I find a bit unclear today: What is the criteria, when adaptive meshing is used? Is it just the presence of `[exclude_object]` along with named objects in the gcode? What if I wanted to not use adaptive meshing? Would I need to turn off naming objects in the slicer or take out `[exclude_object]`?

Reply:
if you wanted to not use adaptive meshing, you would just call BED_MESH_CALIBRATE without the ADAPTIVE=1 flag on the command

Reply:
What I wanted to point out:
* Using `ADAPTIVE=1` is not documented in the Adaptive Mesh Chapter (https://github.com/Klipper3d/klipper/blob/master/docs/Bed_Mesh.md#adaptive-meshes)
* It is somewhat "hidden" documented in the next chapter (https://github.com/Klipper3d/klipper/blob/master/docs/Bed_Mesh.md#calibration)
* It is not documented in https://github.com/Klipper3d/klipper/blob/master/docs/G-Codes.md#bed_mesh_calibrate

Reply:
@Sineos I've just made some updates to the bed_mesh.md for clarification on setup requirements. If this looks okay, I can move forward with making changes to the exclude_object.md about setting up object processing with moonraker.